### ISSUE TITLE
Facebook Feed Api

### ISSUE THREAD

**Issue by JuKu** [JuKu/JuKuCMS#185]
Add a api to get facebook feed.

**Plugin**: facebookfeed

**Comment by JuKu** [comment]
Currently facebook has changed app restrictions, so it's very difficult to create a good plugin to do this. Maybe we should move this issue to icebox.

**Comment by PascalReintjens** [comment]
> Maybe we should move this issue to icebox.

Yes I agree to move this issue into the icebox. A few weeks ago I tried to work with the facebook api, and it seems like they are currently reorganizing it quite a lot, so we should wait with this
